website/static/js/pages/wiki-edit-page.js

```diff
@@ -11,6 +11,7 @@ require('ace-mode-markdown');
 require('ace-ext-language_tools');
 require('addons/wiki/static/ace-markdown-snippets.js');
 
+var Comment = require('../comment.js');
```

Comment:
@sloria <code>js/comment</code>? or still <code>../comment</code>? At least <code>project-dashboard.js</code> is using the former now